lib/index.js

```diff
@@ -51,11 +56,15 @@ function lint(editor) {
     plugins.push(recommended);
   }
 
+  if (styleGuideWithoutConfig) {
+    plugins.push(styleGuide);
```

Comment:
Current set-up is that either the by the user defined (`.remarkrc`) plugins run, or if that isn’t found, the enabled presets in the linter’s settings run.

Because the configuration is added in the `defaultConfig`, the engine overwrites it if there’s configuration found.

So I feel both the setting description and the code are correct?